converters/fromZigbee.js

```diff
@@ -1681,6 +1681,35 @@ const converters = {
             }
         },
     },
+    ts0201_temperature_humidity_alarm: {
+        cluster: 'manuSpecificTuya_2',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const result = {};
+            if (msg.data.hasOwnProperty('alarm_temperature_max')) {
+                result.alarm_temperature_max = msg.data['alarm_temperature_max'];
+            }
+            if (msg.data.hasOwnProperty('alarm_temperature_min')) {
+                result.alarm_temperature_min = msg.data['alarm_temperature_min'];
+            }
+            if (msg.data.hasOwnProperty('alarm_humidity_max')) {
+                result.alarm_humidity_max = msg.data['alarm_humidity_max'];
+            }
+            if (msg.data.hasOwnProperty('alarm_humidity_min')) {
+                result.alarm_humidity_min = msg.data['alarm_humidity_min'];
+            }
+            if (msg.data.hasOwnProperty('alarm_humidity')) {
+                const sensorAlarmLookup = {'0': 'MIN_ALARM_ON', '1': 'MAX_ALARM_ON', '2': 'ALARM_OFF'};
```

Comment:
- What does min/max alarm on mean? (it's unclear to me what it means)
- Please make it lower case
- Adding `alarm` to the property is duplicate, e.g. `ALARM_OFF` -> `off`

Reply:
What does min/max alarm on mean? (it's unclear to me what it means)
===> Alarm will set 1 or 2 when temperature or humidity above/below than max/min
Please make it lower case
===> Okay.
Adding alarm to the property is duplicate, e.g. ALARM_OFF -> off
===> Okay.